File: Iniciante/1035 -Teste de Seleção 1/Python/1035.py
def main():
    A, B, C, D = input().split()
    A = int(A)
    B = int(B)
    C = int(C)
    D = int(D)

    print(validacao(A, B, C, D))


def validacao(a, b, c, d):
    if b > c and d > a and (c + d) > (a + b) and c > 0 and d > 0 and a % 2 == 0:
        return "Valores aceitos"
    else:
        return "Valores nao aceitos"


main()

File: Iniciante/1035 -Teste de Seleção 1/JavaScript/1035.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split(" ");

  const A = parseInt(lines.shift());
  const B = parseInt(lines.shift());
  const C = parseInt(lines.shift());
  const D = parseInt(lines.shift());

  console.log(validacao(A, B, C, D));
}

function validacao(a, b, c, d) {
  if (b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 == 0) {
    return "Valores aceitos";
  } else {
    return "Valores nao aceitos";
  }
}

main();
